### PR TITLE
Bug batching for Scalability and Performance 4.16

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.16

Issue:
N/A

Link to docs preview:
https://77809--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html

QE review:
N/A

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -1779,6 +1779,12 @@ To retain the catalog rebuilding functionality, use `--rebuild-catalog`. However
 [id="ocp-4-16-scalability-and-performance-bug-fixes_{context}"]
 ==== Scalability and performance
 
+* Previously, the Machine Config Operator (MCO) on single-node {product-title} was rendered after the Performance Profile rendered, so the control plane and worker MachineConfigPools were not created at the right time. With this release, the PerformanceProfile renders correctly and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-22095[*OCPBUGS-22095*])
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:1782]:
```suggestion
* Previously, the Machine Config Operator (MCO) on single-node {product-title} was rendered after the Performance Profile rendered, so the control plane and worker machine config pools were not created at the right time. With this release, the Performance Profile renders correctly and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-22095[*OCPBUGS-22095*])
```

Comment (mburke5678) [reply release_notes/ocp-4-16-release-notes.adoc:1782]: Perhaps it doesn't matter, but would it be more accurate:

```suggestion
* Previously, the Performance Profile on single-node {product-title} was rendered before the Machine Config Operator (MCO) was rendered, so the control plane and worker machine config pools were not created at the right time. With this release, the Performance Profile renders correctly and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-22095[*OCPBUGS-22095*])
```

File: release_notes/ocp-4-16-release-notes.adoc [hunk 2]
```diff
@@ -1779,6 +1779,12 @@ To retain the catalog rebuilding functionality, use `--rebuild-catalog`. However
...
+
+* Previously, the TuneD and `irqbalanced` daemons modified the Interrupt Request (IRQ) CPU affinity configuration, which created conflicts in the IRQ CPU affinity configuration and caused unexpected behaviour after a {sno} node restart. With this release, only the `irqbalanced` daemon determines IRQ CPU affinity configuration. (link:https://issues.redhat.com/browse/OCPBUGS-26400[*OCPBUGS-26400*])
```
Comment (mburke5678) [inline release_notes/ocp-4-16-release-notes.adoc:1784]:
```suggestion
* Previously, the TuneD and `irqbalanced` daemons modified the Interrupt Request (IRQ) CPU affinity configuration, which created conflicts in the IRQ CPU affinity configuration and caused unexpected behavior after a {sno} node restart. With this release, only the `irqbalanced` daemon determines IRQ CPU affinity configuration. (link:https://issues.redhat.com/browse/OCPBUGS-26400[*OCPBUGS-26400*])
```

**Comment by mburke5678** [comment]
@obrown1205 A few nits. Otherwise LGTM!

**Review by skopacz1 — APPROVED** [review]
Merge review LGTM

**Pull request merged**